doc/cookbook/source/examples/kernel/sub_sequence_kernel.rst

```diff
@@ -0,0 +1,51 @@
+=========================
+String Subsequence Kernel
+=========================
+
+The String Subsequence Kernel (SSK) is discussed by :cite:`lodhi2002ssk`.
+
+A subsequence is any ordered sequence of :math:`n` characters occurring in
+the text, though not necessarily contiguous.
+
+More formally, string :math:`u` is a subsequence of string :math:`s`, :math:`iff` there
```

Comment:
I know, it is a good point, but for now we want to keep the cookbooks a bit more minimal